javascript-loops/main.js

```diff
@@ -0,0 +1,72 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumberToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber < 11) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+getNumberToTen();
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber < 22) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+getEvenNumbersToTwenty();
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count < times) {
```

Comment:
This should be less than or equal to. You're currently doing 1 less than `times`

Reply:
ahhh yes that makes total sense as count starts at 1, thank you sir!

the sneaky undefined has been destroyed!